sdk/openai/Azure.AI.OpenAI.Assistants/api/Azure.AI.OpenAI.Assistants.netstandard2.0.cs

```diff
@@ -405,6 +475,108 @@ public partial class OpenAIFile : System.ClientModel.Primitives.IJsonModel<Azure
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public OpenAIPageableListOfThreadMessageObject(string value) { throw null; }
+        public static Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject List { get { throw null; } }
+        public bool Equals(Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject left, Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject right) { throw null; }
+        public static implicit operator Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject (string value) { throw null; }
+        public static bool operator !=(Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject left, Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadMessageObject right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct OpenAIPageableListOfThreadRunObject : System.IEquatable<Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public OpenAIPageableListOfThreadRunObject(string value) { throw null; }
+        public static Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject List { get { throw null; } }
+        public bool Equals(Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject left, Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject right) { throw null; }
+        public static implicit operator Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject (string value) { throw null; }
+        public static bool operator !=(Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject left, Azure.AI.OpenAI.Assistants.OpenAIPageableListOfThreadRunObject right) { throw null; }
+        public override string ToString() { throw null; }
+    }
```

Comment:
These pageables should be internal too because we have the `PageableList<T>` class instead, correct?